src/SplitPackages/Program.cs

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace SplitPackages
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Contains("help", StringComparer.OrdinalIgnoreCase) ||
+                args.Length < 3 ||
+                args.Length > 4)
+            {
+                PrintUsage();
+                return;
+            }
+
+            bool whatif = args.Length == 4 && args[3].Equals("whatif", StringComparison.OrdinalIgnoreCase);
```

Comment:
this line is removed, I've updated the tool to use our command line utils in https://github.com/aspnet/DnxTools/pull/17